etcd_funcs.go

```diff
@@ -0,0 +1,135 @@
+/*
+ * File: etcd_funcs.go
+ *
+ * Description: holds the etcd instance structure and funcs
+ */
+
+package main
```

Comment:
Bad habits die hard it seems. I have since moved and adjust most of the code base to better fit a proper OO and golang design.

https://github.com/rbisewski/lclusterd/commit/da87e0ea32046a5fa59563ce711f9a73c2713733
https://github.com/rbisewski/lclusterd/commit/877f35f7663fd44b502b253280693dad5d219495